packages/@okta/vuepress-site/docs/reference/api/inline-hooks/index.md

```diff
@@ -378,19 +378,19 @@ curl -v -X POST \
 
 <ApiOperation method="post" url="/api/v1/inlineHooks/${id}/lifecycle/deactivate" />
 
-##### Request Parameters
+##### Request parameters
 
 | Parameter  | Description                                                                     | Param Type   | DataType                                    | Required |
 | ---------- | ------------------------------------------------------------------------------- | ------------ | ------------------------------------------- | -------- |
 | id         | The ID of the Inline Hook you want to deactivate.                                   | Path         | String                                      | TRUE     |
```

Comment:
no period and insert that

The ID of the Inline Hook that you want to deactivate